src/services/orderbook_service.ts

```diff
@@ -249,15 +250,19 @@ export class OrderBookService {
         // as SQL variables in the "AS" syntax. We leave 99 free for the
         // signedOrders model
         await this._connection
-            .getRepository(PersistentSignedOrderEntity)
+            .getRepository(PersistentSignedOrderV4Entity)
             .save(persistentOrders, { chunk: DB_ORDERS_UPDATE_CHUNK_SIZE });
     }
-    public async splitOrdersByPinningAsync(signedOrders: SignedOrder[]): Promise<PinResult> {
+    public async splitOrdersByPinningAsync(signedOrders: SignedOrderV4[]): Promise<PinResult> {
         return orderUtils.splitOrdersByPinningAsync(this._connection, signedOrders);
     }
-    private async _addOrdersAsync(signedOrders: SignedOrder[], pinned: boolean): Promise<AcceptedOrderResult[]> {
+    private async _addOrdersAsync(signedOrders: SignedOrderV4[], pinned: boolean): Promise<AcceptedOrderResult[]> {
         if (this._meshClient) {
-            const { rejected, accepted } = await this._meshClient.addOrdersAsync(signedOrders, pinned);
+            // TODO(kimpers): FIX TYPES HERE
+            const { rejected, accepted } = await this._meshClient.addOrdersV4Async(
```

Comment:
Just to make sure we are on the same page, `meshClient.addOrdersV4Async` will be changed to accept orders with `verifyingContract` instead of `exchangeAddress`?

Reply:
Yep 👍